modules/internal/ChapelTaskID.chpl

```diff
@@ -30,6 +30,9 @@ module ChapelTaskID {
   inline operator ==(a: chpl_taskID_t, b: chpl_taskID_t)
     return __primitive("==", a, b);
 
+  inline operator !=(a: chpl_taskID_t, b: chpl_taskID_t)
+    return __primitive("!=", a, b);
+
```

Comment:
Is this related to this PR, or just something you noticed and added along the way?

Reply:
IIRC, just something I noticed.